### ISSUE TITLE
Hi

### ISSUE THREAD

**Issue by ed728** [mm11wils/project-file#1]
Seb, be more specific of what you want.

**Comment by mm11wils** [comment]
make an engine I can add stuff to easily :dancer:

**Comment by ed728** [comment]
What do you mean by engine? By specific I mean an actual spec of what it should do and how it is to be used.

**Comment by mm11wils** [comment]
if you look at the code you'll see everything is written into one main piece of code, what we need is to have it separated, so have a module that consist of the Map generation and one of the render to screen and one where all objects are defined and one where keypresses are defined, and any other module that will make it easy to focus on these separate topics within the game.

**Comment by ed728** [comment]
Modular usually means that it is broken up into classes or modules. Different files is just a convenience. So all you want is to copy the code to different files?

**Comment by mm11wils** [comment]
Yes, this would be very helpful, otherwise it ends up looking like the "Main" code i've included.
